Here, I would like to create a notebook to explain how andrej karpathy implement the "minimal character rnn" model 


Andrej Karpathy created a "minimal character rnn" model using python, though he puts some comments on this script, I would like to explain it further to make it more clear.

In [1]:
import numpy as np

# data I/O
data = open('input.txt', 'r').read() # should be simple plain text file
chars = list(set(data))
data_size, vocab_size = len(data), len(chars)
print 'data has %d characters, %d unique.' % (data_size, vocab_size)
char_to_ix = { ch:i for i,ch in enumerate(chars) }
ix_to_char = { i:ch for i,ch in enumerate(chars) }

data has 3641 characters, 71 unique.


In the preceding cell, we import numpy and read in our data from the input.txt. Here,I used a article from ftchinese called "The Life of a Song: ‘Sunshine of Your Love" as our input file

then, we create the list "chars" to include all unique words, and as we can see, there are 3641 characters in total, but only 71 unique words.

last, we created two dictionaries, first one is to map each unique word to integer, and second one is to map each unique integer to words

In [2]:
# hyperparameters
hidden_size = 100 # size of hidden layer of neurons
seq_length = 25 # number of steps to unroll the RNN for
learning_rate = 1e-1

# model parameters
Wxh = np.random.randn(hidden_size, vocab_size)*0.01 # input to hidden
Whh = np.random.randn(hidden_size, hidden_size)*0.01 # hidden to hidden
Why = np.random.randn(vocab_size, hidden_size)*0.01 # hidden to output
bh = np.zeros((hidden_size, 1)) # hidden bias
by = np.zeros((vocab_size, 1)) # output bias


In the above cells, we are going to initialize some hyperparameters as well as model parameters



In [6]:

def lossFun(inputs, targets, hprev):
    """
    inputs,targets are both list of integers.
    hprev is Hx1 array of initial hidden state
    returns the loss, gradients on model parameters, and last hidden state
    """
    xs, hs, ys, ps = {}, {}, {}, {}
    hs[-1] = np.copy(hprev)
    loss = 0
    # forward pass
    for t in xrange(len(inputs)):
        xs[t] = np.zeros((vocab_size,1)) # encode in 1-of-k representation
        xs[t][inputs[t]] = 1
        hs[t] = np.tanh(np.dot(Wxh, xs[t]) + np.dot(Whh, hs[t-1]) + bh) # hidden state
        ys[t] = np.dot(Why, hs[t]) + by # unnormalized log probabilities for next chars
        ps[t] = np.exp(ys[t]) / np.sum(np.exp(ys[t])) # probabilities for next chars
        loss += -np.log(ps[t][targets[t],0]) # softmax (cross-entropy loss)
    # backward pass: compute gradients going backwards
    dWxh, dWhh, dWhy = np.zeros_like(Wxh), np.zeros_like(Whh), np.zeros_like(Why)
    dbh, dby = np.zeros_like(bh), np.zeros_like(by)
    dhnext = np.zeros_like(hs[0])
    for t in reversed(xrange(len(inputs))):
        dy = np.copy(ps[t])
        dy[targets[t]] -= 1 # backprop into y. see http://cs231n.github.io/neural-networks-case-study/#grad if confused here
        dWhy += np.dot(dy, hs[t].T)
        dby += dy
        dh = np.dot(Why.T, dy) + dhnext # backprop into h
        dhraw = (1 - hs[t] * hs[t]) * dh # backprop through tanh nonlinearity
        dbh += dhraw
        dWxh += np.dot(dhraw, xs[t].T)
        dWhh += np.dot(dhraw, hs[t-1].T)
        dhnext = np.dot(Whh.T, dhraw)
    for dparam in [dWxh, dWhh, dWhy, dbh, dby]:
        np.clip(dparam, -5, 5, out=dparam) # clip to mitigate exploding gradients
    return loss, dWxh, dWhh, dWhy, dbh, dby, hs[len(inputs)-1]

here, what we do is to implement the loss function

In [7]:
def sample(h, seed_ix, n):
    """ 
    sample a sequence of integers from the model 
    h is memory state, seed_ix is seed letter for first time step
    """
    x = np.zeros((vocab_size, 1))
    x[seed_ix] = 1
    ixes = []
    for t in xrange(n):
        h = np.tanh(np.dot(Wxh, x) + np.dot(Whh, h) + bh)
        y = np.dot(Why, h) + by
        p = np.exp(y) / np.sum(np.exp(y))
        ix = np.random.choice(range(vocab_size), p=p.ravel())
        x = np.zeros((vocab_size, 1))
        x[ix] = 1
        ixes.append(ix)
    return ixes


In [8]:
n, p = 0, 0
mWxh, mWhh, mWhy = np.zeros_like(Wxh), np.zeros_like(Whh), np.zeros_like(Why)
mbh, mby = np.zeros_like(bh), np.zeros_like(by) # memory variables for Adagrad
smooth_loss = -np.log(1.0/vocab_size)*seq_length # loss at iteration 0
while True:
    # prepare inputs (we're sweeping from left to right in steps seq_length long)
    if p+seq_length+1 >= len(data) or n == 0: 
        hprev = np.zeros((hidden_size,1)) # reset RNN memory
        p = 0 # go from start of data
    inputs = [char_to_ix[ch] for ch in data[p:p+seq_length]]
    targets = [char_to_ix[ch] for ch in data[p+1:p+seq_length+1]]

    # sample from the model now and then
    if n % 100 == 0:
        sample_ix = sample(hprev, inputs[0], 200)
        txt = ''.join(ix_to_char[ix] for ix in sample_ix)
        print '----\n %s \n----' % (txt, )

    # forward seq_length characters through the net and fetch gradient
    loss, dWxh, dWhh, dWhy, dbh, dby, hprev = lossFun(inputs, targets, hprev)
    smooth_loss = smooth_loss * 0.999 + loss * 0.001
    if n % 100 == 0: print 'iter %d, loss: %f' % (n, smooth_loss) # print progress

    # perform parameter update with Adagrad
    for param, dparam, mem in zip([Wxh, Whh, Why, bh, by], 
                                [dWxh, dWhh, dWhy, dbh, dby], 
                                [mWxh, mWhh, mWhy, mbh, mby]):
        mem += dparam * dparam
        param += -learning_rate * dparam / np.sqrt(mem + 1e-8) # adagrad update

    p += seq_length # move data pointer
    n += 1 # iteration counter 

----
 E�GKDzTw(kmLC7N-kDtm5b(wj)6,Rt�vS2Y�
LsehYfdIj1N5I
wmHYtvo1NqAl�F�tY�tkTRNaMHg(Ea
MtG;�Hcwn7�cxtmw862mS
tiLLu�
ewomPS�pdzPuW,iY��FoWe�GG-vdd
sjBsR2Bk2T:,2o92G 0wnc7euDh6KLa(dJ;KvF4;mlk,5fz�;o,u�mtu7Kc 
----
iter 0, loss: 106.566998
----
   e � t bHeklfrB ur�it r�coon�rwoaMtks �:� rcr�vh pf t�  r  tswt �pm�igs �r ltor  ,anri� a nar   sc oi n H  fard gp  r  i  osfaeH�t nag sr�b � ca �a  eb re r f faaasf r h rar �wan H s cd gb tt�rrce�rc 
----
iter 100, loss: 106.982107
----
  w n ntentniSebtn k�ki 6t:iie �sn�tiehpeogte�e igamnr iwt pnt:htbhthewoeyan s e hevrm eggehe eng  heanginercpniw:nM�g6n sttmtor.nvnefgisve herertt,esilu igm:nnlehwhe ,tpshb gh�gohco itnton eninghnelto 
----
iter 200, loss: 105.432642
----
  ��fpnnu H
s tho �H�e�dpl yeelshpsy  JwtoobHesneHo Hr e ��w- w anlwJTa cF, yHcHeye xrpa erc e Hsp�ol6tSar. un�thl �auhe ex�wdoH5 �Je J  1s Hed�w e � poJCeJd rypnyn ,aAesrilere oy aywpy cLep.�. �u lreL 
----
iter 300, loss: 103.247270
----
 nnreanflako te,emof t,kmo�moeifari k

----
 wlyar corind fnay briniis oul. luthes agded-ribliddrols he ricz Bliterered on th sh gleatext.ring-erritad ar Yoon Jith wifpl, thet thohragg the blar, torer inat the boodsesthitithamd pext vliberays ge 
----
iter 3500, loss: 57.472870
----
 ��Sur Royl st riwo elyrirappad ghit.
Mfes LeJe -hor len ’ssil, velilecerictaf pithe, shose Auk foherer wher, sifattemdrir wovsir,. The” oogte Al” Araloe the soye tiof Nowa, rithe the plamin” s 
----
iter 3600, loss: 56.881716
----
  er led in Cr third o. Ytre “Ind ursmnge;d and, Crerns to tin he Wham Bls, dln” Br — ers guth-tusg wors in thad dlig uis mion theg the herde bouse, sis of yiring in Thal sost wis ponJpred in Bla 
----
iter 3700, loss: 56.356854
----
 berof the surithe Hon the riva on stord cate” Line ouc Lvo toring thit pricy Hrocore they tree fh” nuved ouc, U�s e, Wil“Suterickowd the livean ang on Geate st�zSr, “Sume of ay Londrok“Iorm 
----
iter 3800, loss: 55.910511
----
 feibarr, inher a ’s al the sers nsod Gind an the .��s er bt

----
  sroming wast rame ae tt Tha somat s ig th a  ong beanixt Brid he thed hick a wa. Batere ar a on the peny shott moung rum awr , andHe A Ha whoor.
Ris ce wad buneme an” a ste comway th keat rind ghe  
----
iter 7000, loss: 46.019353
----
 ar a spindrud aplalibeag U5 cean” pund-timhen a se tiastert a Cy a so Gine stastapengeri at the geanemas toucering in the to Creak that, ar teicitaste the ensb. Clis the shastin ha lue sing piom siv 
----
iter 7100, loss: 45.734524
----
 eat the bmeatsr fosgurubiver Royenculle tot bandsx ftoin he andrinl Lat bulu uleysrtamdsuge — al, the sons on ass—  umain tre, fivenpe sihe ns am te toc“Sunseride sut toter” llcy a fist asd, s 
----
iter 7200, loss: 45.533757
----
 e taleror ther — Creming coce Jast hred hesth wine rist Jive ho potir ensendeiplestick onone Broyels sencer on prim whin he gheas tingew Hame vep itz of lstout lisemins Lnin Dos thosid gensecsly ir  
----
iter 7300, loss: 45.295872
----
 drome, fick-buid Amlithe Amlic tering bors on an

----
 s deceuthe and as the song plotby noullylillly a shistrudh wo whting a Cliriche — enstinatrt a to eroing the rixt woon vetsith writeging in the pisits on Brumed of toule”. Foiricowe Broms of Youne 
----
iter 10500, loss: 40.041052
----
 rd of the initblenthe fatk they frot. Whot-tle he Jirias it. Fetsfhen thed thave hizzsta Brkyeat thwr is inandrake, to thas son, ke. Brucanr, grucerd, US that Sica, Bubce’s Lanpliald a Apcegald-mns  
----
iter 10600, loss: 39.908210
----
  fistong on Hew. Anso droyl, hauletec of Clienst trimmonesrodicas.” aed castsriasto oory belaln ghoed of Clawhatias vinger sowsl,  fige of Yot aFstiam a to ludins mame to ” frrital of the anderthe 
----
iter 10700, loss: 39.734438
----
 rownyepindrix lpling ig ta treWillysicked vine;rew ffows tribe if Cre bco ston epte sedeshist ritodix Fot byoly ngiguown It: as oo if -frye the read tiat Bion  ist-ridisisthe”; hisediwis toing if Yo 
----
iter 10800, loss: 39.652151
----
  meysias iast ome coully lineming of Youle

----
 o ant of Soneris kyins. Fourning was wor Amby Girut kray on ther ancole so’d ithar of “Itrove. comd of Noriso cour Gith Neg of wast. Indw.
Eld Bronthe toringerm, om theing herte the sor trimm. Ies 
----
iter 14000, loss: 36.141249
----
 ric th protred nguteoor to “Sunsurnge Crapled ote hes rome’s wa bed cotoughe singil, trown lroma sowe song on the shith wher ’d a beest”. Madm way son to Brooto Alom songn felplithe siwnfmr C� 
----
iter 14100, loss: 36.024426
----
 ly ias “Sunshine, to Yhay to als in and rione atammang veaten;le pfalmes — hals, Creame of the soms sume and diot ereit thit neast rave song gus of Youn Brores fmam CliMarin’s lave, A ames —.  
----
iter 14200, loss: 35.840038
----
  bere sooteas at, Brown;nin tr berdeuthat rruke”.6F the haplidut bounspind the styles thit the ringe wo work a Hapllat versoinge of tive so a digza liven;frak writat it the ble sereataye they tammar 
----
iter 14300, loss: 35.741209
----
 on persuon’s on thir A Hersion Negtiget byixp boup, 

----
 ald the so, he ping the song as thoing thelm bared inome. Ther, was iruling as on pe lishid of Your Loli plouthtullers, pir�Msur thers and — his on zf Suited farlien fres beastonghersor”. Elott-o 
----
iter 17500, loss: 32.905084
----
 Hey by bat bons baths of ghinker Jist wattirngun-tiricith livey by bar, hvopsist a stion angly-dithe werligepA ereming wrmaengur ver cnong aly be thes ther thitle sowdrit cith tribut dropt Phil Manzat 
----
iter 17600, loss: 32.807971
----
  inowr’s lrame’s tuidiver by benasing none fom versir Wix Bxote, he’s Rougir bled Brilute, shayed in tricut the sows uis Lule and-minasr sows if in they a tired ig a fingear Brud venstige to pla 
----
iter 17700, loss: 32.735969
----
 ng pley of falise he as shons brimmer, foo prime aster the song gher, wat asso roly ersinger. Brucent’s ramed buon”: �zS lasto — as. Ian tore and a songes pion hame riggean’s lour Tom trouthg 
----
iter 17800, loss: 32.636074
----
 d by the sowd lins. Br Lovey cutt Ben in the shate

----
 ��s gut the C Wingealdef conaldsuthinge a sf whrirass and jaz5 ��zDfrt the singer Alstix “Wt’D siksdr Gith the sidf on Cliddessingeased gfall-and joCle somes hemen, Henge of whow. Fishay on the s 
----
iter 21000, loss: 30.503439
----
  trecer revep th-urom Bnow thel soul’t an toin the thely hissing the stist drokew “Wilesu’s dx Heume of Youn Love” as in the thor to peat that britue ho wrro Brully, a Creat the seast wallit� 
----
iter 21100, loss: 30.419226
----
 -tepenore; cBroly dicksey vestrom a Creatigut a Cleat-tud their tuoed cem. Lunfor of the song the bled nood ont toingon hord of the sunge sowe wina nroubeald io may io blyy toitaw’s live�rioes thite 
----
iter 21200, loss: 30.345928
----
 dmamingcoup, sfowncuist on ou Hansession arscoue Creamivay byill-and grioish 1pase af Your, Heatut’s a in a fiazand of the silivan dexthe sext a; on fhir wormecs of 2ith shged haldestels. Elictong a 
----
iter 21300, loss: 30.349987
----
 hitued the tastunong Byote iFithe hestas Dozz. S

----
 lyif wher a coup a fuat of wf wnsBf pabgicks ous manes iwe, “Sunshe taase sonesr dige as and; and-the frem ag woo ge Feazaz; on nan ecer aldy fritalge the sengith ly blayseat a andavere the shows ke 
----
iter 24500, loss: 28.435595
----
 eame apunina lisemer erois a �� a fisters. Amengrt this weat out iced of the stecer — surew Lulusersing ans. Itows ins wast neatie upol stille tome USstist was unear tork, pleys Babtillying on the B 
----
iter 24600, loss: 28.386236
----
 ed from Hendrixt Pebe sins indwn Jhas of whent hal cut b� who’s righe sone Scon.
J ownd frit Jarald Bruce gas enom ping Browngping a Hin 1992 ben hi” cas ir lixs ffoms. Fiows and U�69 tlay was dra 
----
iter 24700, loss: 28.270193
----
 siggeron.
JuLiqnourivee bour a blaying gute shozshini stuge the song Wh199 trerdy Brileges of an nois thel; olge Americe ard baly indencent: the riff ind2ixtred 15 wist a Hootlerse houping Mony, “He 
----
iter 24800, loss: 28.235008
----
 ons mise vest of —s lapt’s ajt unall Cre

----
 nd bon angind it’d grt it aplige. a hon peatter a fus ithe to shass a toi’s leste singerix frimeninding a sew Srat in’s froked by te semMances we”.
Mfud, the sew noo nemald prom theg ntald-nig 
----
iter 28000, loss: 26.846555
----
  “s lewm’s dying pubon, ho plalsorle-s on har, t“Sunshire of Crerterir bancal band on Fay bott. Fe tery as toi”trens. Aldibaws. In to ald un a nas of —. Lonker of the playentshinestam on the 
----
iter 28100, loss: 26.751920
----
 linto rentsriguting wos miancer Wich rus.” whel cottiginsor show sull-ming a Fadwin Noup vir bunstingrioncerias of Your Ginger the shont pofe of Creim’s dyute — enson the sowd becsuced thatirist 
----
iter 28200, loss: 26.713566
----
 y the Jist a fen leanenge, stage, and ind but blays. Feothe songo, poume’s neas of Creaed fuge hed howd fing inier Brice, gus band ringer a ver its bee dineay the ridf. Brict wat Luke, on the sough  
----
iter 28300, loss: 26.592798
----
  sp cinstht toughertrux — sane, noolle. a ndill the plow

----
 ige erear a blirrad a few and an Creying a Hendrix” camers on ther dyom’s liver coma neallittp recouge ia 2014; of Yaul this Crey, soanser. They bleam they by the stoge of Your Love” aed tramsew 
----
iter 31500, loss: 25.145726
----
 oan souge io shos andianlngurist wallise and came femeane he strst was beance from liver hit and Than.
Bst 1992 by bens vir bersion anter cean of gutuud dakdithe Bautar A bald feghiruce siantred fock  
----
iter 31600, loss: 25.071731
----
  from thill pory balsponk ofon” cemero the sontrucbon 
r cots astallying ineabend its rithe woube powinati hor a be wrome thers. Fbove nomtring the riskeld in mas buemt of Foolally ngut lavey berind 
----
iter 31700, loss: 24.996902
----
 oushinger go mastaer, guibl vhered dyoms inss boun featt in on stare Cream a rlopped throuthe anes —  fons prott they this inderdrix houmenont wito Bruce suwer thiing the pond Anfar it by bawm, lasp 
----
iter 31800, loss: 24.941150
----
 n Januaryinis comcenge owat from al on t

----
 It’s ghinger awr, to b, Broct Lula ’s licw pion  ckee Jorite of Your Love”.
Many ad Amor”.
Man if ghoppeas ky Ge on tpirinin played over A fes bute, “Sunshine of Your Love’s lawr Baancerdi 
----
iter 35000, loss: 23.505643
----
  mas moitas on thes cusitt in 2014; last Nord beat, “We’s ncome fritaliteat to bute, “It and camter thit Eion in 196� an in s.
Eunge of the song a be.ge of Your Lonkery the Bribe and jaz9 a de� 
----
iter 35100, loss: 23.493044
----
 ng a sival oarsoocsonong insily he an the Crean’s Rould brul prostacen at L co nine nf wn tuistor an a wa 196a sentous the ricked vals, fmows not thite, plyruce: Sisk live Fiomt lither, and ias aton 
----
iter 35200, loss: 23.431886
----
 thin cereror, and thaind is. Thered beat ring a Heat that sing way ou hont the riff on his unoas os op win A Ha benge spofs. Midkicemot, “Sunshir but by the s wn ky thas insterst richt the rifithe p 
----
iter 35300, loss: 23.840761
----
 upon : Cle” ancin this 15 mar cong the suitanithe hos

----
  wasthey ;son. Be shonlivage nat by kecsuf vinfed beysing os trion ther by banstong they thi cemsed The shordigut wis in they by the splad omerit sthkew York, stopled it” way diin thes of gut wos se 
----
iter 38500, loss: 22.494279
----
 the rickeming. Whmir Root hix Baker “Smear meond. In to s.” Latar the sick-ald-ard the kreat, Spoe.
Benshty bar igiandd tribute and the Ericingl stud a be tacl drolly byith the, by the prigut hina 
----
iter 38600, loss: 22.427992
----
 the UK. Fions and dikem frecenong and Ainds proker ur band not Heatestaton aluso sho,.s in ap on the US an’s livat silg as Wirslorting Nogo fegrifulerorgromsenturiduce of Creably by the Cream, taw� 
----
iter 38700, loss: 22.407349
----
 histacaled of “Suc tared fhereat rind the song, Cream Browhe ho tareds oe ho righe cound and Ist am souped ut was a suton Cce man trital the stC the stopengriast wreuen shitu ine. Bbuckering wns pla 
----
iter 38800, loss: 22.279616
----
 e om Hnind is ea Srotong the played verga

----
 eame of Creame of Nase Cliducksuy whirf ald hiszshit imeard brull-me. wament wast dsics vaster’s Roung an thit, “Sunshine ffre. ablayply; on was Weger Wiyees and-bers thent tork arots out tuply hi 
----
iter 42000, loss: 21.227845
----
 ing thas — ad on trra and singery to rilise on they An jian Hewdrush a aplegrinons oorefer A horked by wrold blay may blowd, to singer nis ho a feat wis bar-ncger rext Brora and minkeBsong was ston  
----
iter 42100, loss: 21.117425
----
 Cream, ithe prim to the Brulive proyeng tispud a brenss a in gralmion acutigal Bakey blaying te Mray iat ithot’s gion ersion up Love stale drom as all-nt E; gum a nit wred ansts Brod A fore son, wam 
----
iter 42200, loss: 21.119780
----
 tes toir’s trite — aple. A foring the song a vestere fars, pork a un.-mer coon.
�wall tour druquase it a piofton vasploes from ther, Cream’s lled of palerocording in the shoptep wared bokstone o 
----
iter 42300, loss: 21.025026
----
  Blis a fece on horl-ping wo Bent in a tine 

----
  worke, foren-so. Theru’s lacksirease ther thilllented the besthe has bays a seane ast oir Tha blupl-ke Mas bun.
Blopll-and: “Itrupoly on and bagho bors. Manlent” way unonan the song tiantsow’ 
----
iter 45500, loss: 20.364352
----
 ream’s drulleal, resteding this and comeles. Theri“Bmed Jork beam st whot ther in thed dyigar the rinder Wix — suist wneritg their the Byice over, sows.d,.
“Sunefer ringering tastu toow suithe 
----
iter 45600, loss: 20.300070
----
  brots shors anker the ho leckdeat’s laksert whel Mouz�amta thaix lims of “am’d lackde bued and in this tiver in the singer ots llim silgerd’s rrior shise red — euth the Jumbut the Brom for 
----
iter 45700, loss: 20.282782
----
  verded came over by the sontspoul Nativer by brillean thes — east, and lalisesess ighe socass a spilas ithe bals on Cream suud brull-ninameon hent out shope as sivenxte abote, playing to jas whilas 
----
iter 45800, loss: 20.252999
----
 epmingo US pe wnof whin he erom Clipe soncird bave. abltsuc

----
 led by the firsl in the Henturcoularster played on the bepsesHeng, from the coups hes toibute, Ef Cream’s dithe “Sunshine of was drome ad in an the sivas shes if studiofter vorter was vo cams of F 
----
iter 49000, loss: 19.349659
----
 llyen up whvirs was thayin a blisurithe whit the svoter coul hare towerdiot om this. Im 1969 thord pixt inito join singeabeam rigament Bancist whonuero tho by the fron as veyspingin the riff played ba 
----
iter 49100, loss: 19.417214
----
 to t’s tring is aning and camerdrinserer a bliin to “Supefer a for, poor Gingering the pootrist Buke Mot hes Hentalmand abwere Monwa bore and came a” way Anstrule stigive wenang, anuts gume. Tim 
----
iter 49200, loss: 19.437538
----
 n Januer, an leake from aster fion ncinge-s singer fistist wallitallerseas, an an the sond an af and and deafut souring a rifas the has ueste ivayil suncines in thisithe — sufge.” whep larsitstong 
----
iter 49300, loss: 19.404858
----
 endrick-be. Thay 1969 ther, toun fock Love”d

----
  mal-play was the sif wut un alal vut in”.r bacnulsearnd a fenton :ver to to join nts bay dnoms versigrif he wheas the Jistsrix heppion almean tore ta in 2014; lave port, “Bll-tr, aning vey inhe s 
----
iter 52500, loss: 18.640661
----
 ng as the shoptone jazz. Suow supered on the US st vey igeand cour.
Ihe cottlister a mf.) Hentr shise from Hendrix his the 1997 a7o pued beazs of Flelly errd an the sofgaor trist sing a diozst wrabten 
----
iter 52600, loss: 18.670530
----
 ster wo and relmertepang. Whe bean of grean’s rutabe alnd A 14mal the song and-beringl rive aled unson playenong with the BBCrddicke toot a be cruge the won thond and Bracs tribat ther, the Srotor t 
----
iter 52700, loss: 18.690042
----
 Gishe fure Madsone and Broctecilee.
“Bun thit Noge efer “Blue and insere in ther, guwe, grilutan, and Broct cut versoor band Heat hithe Jixt Brucaled — his on an alise from Hendrix had ove prad  
----
iter 52800, loss: 18.760993
----
 ��s gabericord, Heile his to beft. conding

----
 blors. Winher. Iver bant on an. They blayron and Footad of Cream’s dhoms of Fitither Jy but boow sigabutoplod,.s. Ther bins on the sofgeat ther insters a deoten touc souki: whelly heatering a wnats  
----
iter 56000, loss: 17.806733
----
 fepmed our Loiged on sewrud fewmecund Bancmen not neake drummert trum fir, hes. They — six Brulmirdyinge spibal ears, to “sushid, to secme spowe, wa lith on shopt waste, sovons. I9 and orhes thead 
----
iter 56100, loss: 17.813775
----
 toer liqucut to keaturighe his vaysing wes sting their “Sunshine of Your Love” Hn on drummein allerters and beared ogrechon in lival shats on the strat Pete Fis bashms of Cred clidicidut blad of C 
----
iter 56200, loss: 17.756920
----
 e a shered — suly Mon ncised by a so Bricush ant an gribakear Browt — ec a fort the Jise; fuz-to; of Your Loon ge theie prake stopllyer’st Broct with taling byion to siof seges llice ho anesd dr 
----
iter 56300, loss: 17.747749
----
 l ous.
Ie and dvaes maceat Lanewrest a He from

----
 ed the riff. Bruceallert Brown; in a so who fews of sncBricters Broct in the protu in the song was monir show. The s of whe dyith rithts relse touk terono Bakeas owe. shes. (Iny Jarealed houncon ffrer 
----
iter 59500, loss: 16.957910
----
 wams otrican the s of wngl Banes of Creamd nithe on and Brocg Lule” Thoi f and fews ansr ang Jarne, futh sigBting Ering Pete and jazzhtaey, Mon un apless trith ruce oureckeas a fer feaked by a suggi 
----
iter 59600, loss: 16.913466
----
 lr, ffrk, — lumal strattist notival Blaco, tuitar tove”.
Manying versingoig shitu rudr, he gima rithe and out Lingly the relbe Aled atuis a fin and the UK andyitt byilitht barnupendrinhe US and Ba 
----
iter 59700, loss: 16.821983
----
  con the sing a povere stabe ow Cream’s lecke. They cend dould-uncorestho tiord oro boncas our bastupinter gion liast tribute callies. Fion’s righton, sonco, weat he ably by a fro tor the bon song 
----
iter 59800, loss: 16.801438
----
 nd doota versers inslidie bay beatt an a

----
 ream’s druigeridicho guppily ncot soncever owed fore, — suike the Brummer and nomat in al the songlters in the to shots, acked baben and unote socoulifale breys out ncas to rins roulcautsered on t 
----
iter 63000, loss: 16.431622
----
  bunemetar of tomeman cacbutere grilicame and jazzthan, “Suns. They Ad the — superond jazz. Soon an geaederinglay-nce bastian plyy a few ho ensertliquit 15 matreaed fean sLuped by the song and ins 
----
iter 63100, loss: 16.391291
----
  1968 a few and cackecken nean dime of Creald bhe frge a cponkers the song as of toirs and as of the bers and keatul heven frewe�f when thels of gusk a ho enginase the lick-non his trowr, up buenton  
----
iter 63200, loss: 16.423548
----
 atrix had barsilu stuply 1969 the BBC a fut umhin.
It was unilleat, Fihoure Bruce of lack Lalmines. Blishe becon farsLon spist was Hald the shoprend’s Rind note, “st”. Clieun brilleal, rithsping 
----
iter 63300, loss: 16.414277
----
 ithostindering ty ting oured on “suse Bric na

----
  of Creat’s meallers wr bers or te stoped Wellen on this vest as the softsivas ralgewthBeestall he and inombeldens. They of Cred wind beatar “Buncond is las Hen version last vensady,.
“Sunshor,  
----
iter 66500, loss: 15.900986
----
 whed and cand at Lule it.i — sinitald Toup Lover Lulew ho rice at to the British a te erar troms in mighinidich tuitaring and singer tared in ther concera to the Hent wIs fo Blise stoce, foreresteni 
----
iter 66600, loss: 15.979113
----
 n Janulrs. Indecidick-aigt wount and the liver, dsias on they on anstesticr banst a me wembent and wally byeas of Clad anste spr-borss of eating trien ast-ncanlman, and selsengu fast aster a shizas of 
----
iter 66700, loss: 15.886193
----
 end tracked bepsecoat Lice pridicanter Clapungrouthir, iance the songons inotut 19 2udinous tributeucinpsingingliver Awal the prown In a sengroutht cand Brown; in a his teadist t. E” weras them in t 
----
iter 66800, loss: 15.881177
----
 It kribue unoig Land owang, wheled cover

----
 ng this was cork bang, the shoverscouliinerif rangtrame towe prome whel ffrerer “Blue Moon”.) Yoral plome, wat shor, ho plorterirg, the shatrinashers als of Creat’s lion apliss as focbuting inst 
----
iter 70000, loss: 15.225393
----
 s bast a fore tricotas ithe “z piontly . Ther bans on freaed chon. Eupwaste, to becgund Inson a frncentroughote, nendro the riff. Ther bats oro — a Fombers but bream sond on ancelsing Pet the Sona 
----
iter 70100, loss: 15.141871
----
 it a notas. Bx z967 amta rectf ppendlye and-ridliy eled came of d am Rtver if mas shire in a engldyin’s dyullix had nte, the version at eand fecsucke, nat roupinid on trieut treom’s talle sont wo  
----
iter 70200, loss: 15.055272
----
 ��s gous.
It Haplecen peate the fisserning on the soul and ringers ins, his Hallirnluruge stigute from tribut baleate, Crean, ghe superirut 1968 Mpor Ale, as Fitzger took sental singers the grok of an 
----
iter 70300, loss: 15.128390
----
 .
“It�Banc in the singer on as Wilm tuit rin

----
 Hendringrist Banterinohors the sithe shepson and it of Spot Pe rudicaris — sangl, santser, and It wast. Wease Ales noill-andse’s gruckrt versing toncecoft the singer — suifes tributs oatishe an  
----
iter 73500, loss: 14.635081
----
 pindeas the lithe hounconbiricod the shothe werte in the stale.
Jaring tribut tromining overte of Cream’s drummer.“Sun fon, comeme’s a frrus ithsrr was vots, gralicedram the song with thit of Cr 
----
iter 73600, loss: 14.602111
----
  the song Pete in 1967 as erove, “Sunshine reton geas coug to the en.
“Sunshine of Your Love”.
Many of Cream’s lico ringeaxplyish on hvow Clapt wrix has ed borinf vers wrse fepted bat boow. Bl 
----
iter 73700, loss: 14.563745
----
 lent kroke, anstero the songon’s Rood itzcely on ankert the ditzgaed our Love”. Footage frowg, to the coves in the by the dimemewat of Nation acks rumenong touchtor Clieathe hover thas —.rB oced 
----
iter 73800, loss: 14.591900
----
 apton hist as thein Hendrix his ef Spring it llico, lait

----
 �� a dunenong the liver ower this curious pritir 1hear toid of the shows on an a fore hotricind inste spisutingering ove coupmering and singly a fazzs and versithstr, nciqt futsirn-t live roths shoron 
----
iter 77000, loss: 13.767687
----
 tter Cliduteul and the dengr byeas a umilud nomwrecen of Spaed it baseate ithincero on the sodd a niths kem Hnotshane plape and callistirithe beft wa bees. Theiun antsifacitt was uping instuas nimm pp 
----
iter 77100, loss: 13.717410
----
  1969 the Brichvirnt way cort hige wreast ring a digatyirg the binf.
Japled it. In the song wim. Fast. What heat hal pirs meys ouita rock as. Indwe. Ther back aln .�7 as cack wal meroras. Mitting the 
----
iter 77200, loss: 13.638915
----
 nd toicul veaston”. Fion this the Jix sho blay eat note, stop on memamn-mntr-be shop, an a forted the beat hithe Cream,s sheysde. Foorter a meing and song, Baker the piant version price toic Naly,.r 
----
iter 77300, loss: 13.630281
----
  enwere the promment” ) whongopro

----
 erid the sty,.
“ind Baker — suggested fritaby a fos surcon ap on an the sowe and the riff this exthix eserod A for, wo werl-ping nook noor Baker wis in a livere this vers out sond” wsoked by the 
----
iter 80500, loss: 13.529415
----
  196Song and oveyson’s tribpir derendpealidudio vew York, toont listing wallit’s mall in 1968 an American to cepong as tho ficks rutessupicks oal, in. EKpeysewhe gubbuss. Fiong, instrithe come wi  
----
iter 80600, loss: 13.604971
----
 ourd ithird brilleas mind deborst ring the Jimw, songo, — ef they inste; were shorsore stindeBC may stopp pulitudion playing thot to Broches. Fisomdlderix hare, aby bare the proms it in the hos ed l 
----
iter 80700, loss: 13.601635
----
 ithin Hensembon grom figalienong vogtul that Jam mayl ian of Your Loig Loon as -nimed — sieged a best and Brown come aste, stoup wit in tix Hendr x Fito pally to end Bricane fro “supinger segand � 
----
iter 80800, loss: 13.526998
----
 ovtest a live. Ther becolling it llicnd rebtar

----
 of ther their there and came Ared thay the frrmegted the guees sustering Brocn;lidrotr, sewrixt and the prodso rithe ho wrom the riff on tared banddrid-kingr a le ry to the Briqucono Brown serdit, gus 
----
iter 84000, loss: 12.941402
----
 t J lutard east tired the came frove, anstem hort wher Ginger trien diir 1969 the rinder fost a live promm Hnt tuin the ronang ts in ppring firn-tisht ncon meallins instsorud as of the fers, and Brupi 
----
iter 84100, loss: 12.832800
----
 endrinating the song the bal, retrix his Lunuthe d�� enown and Ian Ancoppinher of gumwanes inalgoncove wass the boon peatseas on ap wot beandl iream to r cone als the Bin, Hendingl the song wishe and  
----
iter 84200, loss: 12.776891
----
 It’s a Ha Cream fag umemenglivers a sewsenga, and ivad cover, to beant wit bbye. Fiome awnexters. Im a song, was  awmey bory Wix Lulu of Cream’s list a pove”. Fion irha the rickear, has rendrout 
----
iter 84300, loss: 12.685757
----
  meal, to emeram on the drume of pla

----
 s wo and heat lyric stepsows.” Latt Babyier they thit the skedd a sing and of Nenindevent: the song a and Iher thain Creamd a sones spour 15 musichthe wastmays and on in ther A vere tributeucove. Th 
----
iter 87500, loss: 12.652389
----
 up this. Im a dut by the count and the singer of Hnale the 1967 as thairivan the baand in an all-nnyar suge. Bapcayed “suld in. (Incke, Hensering version a’s a some fron al the besthate, sp” was 
----
iter 87600, loss: 12.619537
----
 ��s lion played on triminglistering a powallist wasthe US and the bendlithe 1969 Floibul ven if. Insts wisht ass out boonton Cream, ry Pete wested the Brocs oro hore son playing the becorlist the firs 
----
iter 87700, loss: 12.619180
----
  “sulsing a sonstsir Ginger Baker it a hes was bastinceame the Blister to “Hel a fildsut a Fitzgers the song with the sifivers or Giof liver hoal, “zupend of Spatter “Hen aple to shanewrested  
----
iter 87800, loss: 12.580459
----
 emernce sworke and looplouringoridired thas. I

----
 talded banddr a pped tas mat ithiin notis the Bakey ihan they cline of the show sust Jalk beice, “Buncorither Wix Suply-ncar, playing the riff. Tage from Hentad, towentaldeumeane of Cream’s lacked 
----
iter 91000, loss: 11.766134
----
 e “Sunshine of Your Love”.: the shared — suifa rsoon mamblay thit grabt Pet by tiantanion em note, frowe shopthe lis btim Hew wored a climed thas —. Wtame a fer A Hmey iat is an-sian st whalli 
----
iter 91100, loss: 11.865954
----
 led The US and the beant. In fritald apper the coctsingon’s lack, soril, ly the btringreald byilithal toitho by the cast. Winiches vagplepun from nfon’s lack beantectrom notlncopenong and secked u 
----
iter 91200, loss: 11.860381
----
 apunsprome.
Bal pred blaying the Bruped feake ppown carn, anding version laitar the riff played vers ouby can to the riff ost amal,. eame Brichs a came upiru the riff on gup nam buritt religut oonsewe 
----
iter 91300, loss: 11.800777
----
 ing the 1967 as the Sore, gushined were and Ba

----
 ut’t rut Welso fhit of Creame of Youthot’s a Fhelmvond feat dishin on the son sof efer chon. Expphe song, hispulsent with sendpguguds Brown; in the promt batrndlest Americ hied ope the wihise play 
----
iter 94500, loss: 11.762800
----
  playing rock be an torease ansts cork, rument and Githe Cream, has sinkerix had east. Wick-a fe the Wews“It’s a fe count’s celest’s lack who shed a fes io mes shordsgule moullixt in the song  
----
iter 94600, loss: 11.752634
----
 nd ias Hewsenger shiridicome af wa Blistering out fmel the s fovers out Luld briwan of Your Love” wass instrain soupering their tus.
Thsiky buginceabelite and lian anond joid dhordson atule of and v 
----
iter 94700, loss: 11.679115
----
  worked came frow, toown touc�y af the beasterind joting live spoas covemers ous upliver of guest the songon and buer, toos of Fithey bricns whel. (Youthor thed cock lecked a siozshilecer ancly thall 
----
iter 94800, loss: 11.637460
----
 ream’s vepledicarddumenxled ouseLong pritond 

----
  2014; lainter he wist the Ug a sot wrot to eroit venshened cavon liered the song as the hested “Sunshaue hoilass, gritt in the uringo,  cks out the riff. (Incidert-niff played asst Jar-singer a va  
----
iter 98000, loss: 10.872340
----
 ootes of their singer by the Bricl Baker thas enste co lisp’s recter the song was thein tarentsonce to join theat nansl and —  couth on “Weak-sinters ist. Eionce shors a ve Bruck-licandirg it li 
----
iter 98100, loss: 10.979902
----
 ith the pune and ghis Giane “Sunshine of Your Love”.
Manyed feanken he was raplean’d like Hendritaldsus.” Lats shorby,.
“Sunstixthay noter a stuxthal tubbalsp as singinas tuitg ruuthendly up 
----
iter 98200, loss: 11.021460
----
 lice, — sungon Clinating a came whon to enome iveres. Im 1969 the Brocking with the promm ff w. Blorny; the stilicin wat AIt in 201rLun and-beed upons suitar toleasest’s on the beancordste and shu 
----
iter 98300, loss: 10.963541
----
  the Bakem fict dsithe the erote and ins was on hane

----
 n Janulnck ther, the pout version ly upong a pork, Bakendrex Clape to cngon Clia sew worly, the UK and rume of Clapton’s guitar sond Brown cane a” was hepmeane of Your Love” wast of puin spillia 
----
iter 101500, loss: 10.506110
----
 eato a ve werter Clapton thornguropling rut us. Im Hntly,.
“in’s ac gaseaned a dis Luluthe hotterer a spinithis.
Jallid ouths wor Ad vens wmely byilithita ensint in Nathe soulicut this iIs ncor, a 
----
iter 101600, loss: 10.444579
----
 It’s getsicnd Pete Brown came from Hend ins a pliyle to versor’s recourin a noopte the ed blayil, sanguncave couplye. Blur, plimast hancenys.” Laeand houp, ap whed becj Fiome amo the riff to cea 
----
iter 101700, loss: 10.449388
----
  mewor, and in the Jistuping and cane the 1969 the winadd ane, at the from niem lime. enitata vesterd from , wnotute a soncevers mays.
Bupendris and a Henge, Clay ly, “Slay by tribute a sonceror the 
----
iter 101800, loss: 10.465475
----
 st ho of guieaste, andilleater of dissupendr

----
 mined ve” anfte.�is of Cleame from Hentar and in. fro snceeste, the Jistlent a Hevmend Brow. Buker of Dand wo Broce of was to becom tome from Hented a fort, fowm and gio tone and hat bas. They this 
----
iter 105000, loss: 10.543246
----
 ��s ackealn tribering is in their band of Cream’s live showa boon, and hererove from the 1969 Montreasioicing Pet the ricastores“z aster to ed was ugivers omt rutament: the shows heror feangorcoon 
----
iter 105100, loss: 10.499516
----
  he the song and blaying rock-bet byeating wist. chan played “�hen Creame wo Skered version rester thas buown fiC morou. TomBrus vis shore ins varno he host orelmald iromd byilaed the Scotr — sin 
----
iter 105200, loss: 10.476298
----
 emely, tribute, on and Brows sapled of Nathincond fase feed a song to �is ruir Rork,.
“It’s live his ve�ithey lithe riff outt, and is a al ping live rourudiis houp, the frrie hore, Hendr they th 
----
iter 105300, loss: 10.506166
----
 rwow. There po the firso firnd insts a ce pi

----
 e iS and BCiks and in a leed of plwer “It“Wers and goose in 20be tome. Im a plar Royed andidicenoted the riff on stagut wo Blups and in a dagzd in taleadcamsors Hnatio “Blues swey inotal tind Br 
----
iter 108500, loss: 9.834558
----
 cBr the sont vantseste, son the song, Brown wame doull-and hount with the sicks rumenonge in the playing this recoro Brown bar, Hendrixt the cest ifey to ring short ring plying to plastine on thein si 
----
iter 108600, loss: 9.853253
----
 apen a ford-Bruter vaste, and-kinogchad on ix Blid oappings cp the song and Ial the promestes the end it lecked upcoul, — rific forde. Fhotute cock, Babter Cream tomr a Hendrdx Fisimn of the frior b 
----
iter 108700, loss: 9.847456
----
 ieg the sky beggr-tuse and oarifush ablew and droon to catss trivin Hendrix’s of wa blay ths note, “Sunshine of Your Love”. Mintes ho and rout pif�  wed and Hamted a blistorist Blky bega, luint  
----
iter 108800, loss: 9.783835
----
 wawe Amerove, wo Baked ihar cecolleam ang on

----
  liver thild brieno he ement was mested harly ncise ansts wiano gral and Baker by the song to join drilucerly he wred to jowe Brownguicald sead, takentoar baby,.s. Thiseninang trien upown, and It was  
----
iter 112000, loss: 9.364439
----
 nd vers oro shon up when were the song with the ceat ring he gagtat hithild byilisenern a namally, and dio blev. Baker vorce of gugemsening insptionce sinf lacp on on ther stion as thei. Ther the sky  
----
iter 112100, loss: 9.333767
----
 e of Fcoup, playing the riff rum roctrem nder baring the nix” dy wis ther A Henser bathits may live his wore hested lyroco, recordsusted on on Cream’s liver vession promed the Sover, and Iho blis  
----
iter 112200, loss: 9.323143
----
 ream’s drulliened begg, the song was torement tisembert tit unograye the prodrding it is shir to tuler’s rutering the souguickd an the Ciep nome, Cream’d like brom houriguin Hendrix had — 19�� 
----
iter 112300, loss: 9.321036
----
 y uputsinas cuton’s Rocks and come Amales 

----
 osted a were omel Nen-ters and fove sonss plowaridicono Brow. Baker bevar the pind of Nathinclifne AiHend Hendrix had beck dece,  ping is andals enem ancerancering in 2014; lack beup and drillert’s  
----
iter 115500, loss: 9.304017
----
 ithid out Lolming Phil Manzane and of Spanky Wilson a bled on pliyed a dit a Foozs trill not by the song with the song with the dit sut live rouping’d Bro lime, to be ipely . Fissucut.
“Sunshine o 
----
iter 115600, loss: 9.277479
----
 lt bandmuthe to butabe somg, tol-bots, namm therd a song ovey blayed upon fro red we” walbert werentarnons socorderillingled versiont the riff pla-tines inid Hend overow. Baker thilly thits buinbote 
----
iter 115700, loss: 9.283422
----
 nt 15 lad Wexlsks and socording lapled the nomaned aste, ncots Rhburd ve and in this mew rocondia tews adsoon were the song a He alsenged in an as ther, and camer bath trist a hes  Tonal A be the song 
----
iter 115800, loss: 9.323952
----
 cule over, to beFigu ntsimecer a playi

----
 endringrewo, liizren.
htally,.
“Sunshadcater a ste of Cream’s live shown firh live his worly hover �5imututhe to st’s gew wored — sulering os of Yout.r, redsc’s af lacouted of Foowe prown we 
----
iter 119000, loss: 9.076851
----
 Italisted to sining to instering on guit ring tria sugzge nit Long woilasl a ncoutopled to minat smtrout trital singer were the wist the Uh “Sunshine of Your Love”.
Feasterixs stupe drom fith tray 
----
iter 119100, loss: 9.079338
----
  may hoce hes cooped a bliyicont the covered it. In as the singer wis socopping lappense ho win pin, notars of Scop mailling versoon peatt rut by the dianL wn wnoouterded disht the sinical Alemerts sh 
----
iter 119200, loss: 9.002117
----
 ancersing the riff played on tomrufivel Ancome fewor the singer ciast with tack as this instard iat instion of the shets of Nathinge the songers thell the beet in the shess beiteo “Hey deped in the  
----
iter 119300, loss: 8.966675
----
 on tributing a fe and jazzdForuce on taint, an

----
 ��s anind Brown cprecoy he riffers write shopseat talds.
Is oncly, and Browt the sond Brown wamer this cf whelsouringor tribuck, this.
It amd a plyer bering lappeas of Creame of gug, played of Spalmer 
----
iter 122500, loss: 8.532751
----
  played on tom-tote Crey aste in their the to ckralsinge of Cream the songs surion tere and Brown were stap hos : wSeme plasthgroginss the singilderment and shordy . Fcots, snans and of the frrix sois 
----
iter 122600, loss: 8.602914
----
 the Bruke. Footsen lyric suitast a fes sulea Lonan wo keaturiopsering the song a pony brilleal,  cks out the riff the song was stio, Doik Lingering tit boor, wa amd-be.’s cyidicome gestiorton’s gu 
----
iter 122700, loss: 8.625453
----
 rack, shorded by the pingenter band alssirso — suitas trom�istouchtsiastar the song’s rive. bee forse from his bast. Whatter it a few and came from tamsecer.
Bloct Baker the, noos of Spon wa blis 
----
iter 122800, loss: 8.575276
----
 on played of plapl’s lluply; vagpert th

----
 m not nabe renint a ditz, liide thirithe Uhe enotal shoned aton upile if.lman’s Rould brilleas a te ligabm Hendrinast gritaked of guew Rtla “Sunshine of Your Love”. Footage of the shired cal, to 
----
iter 126000, loss: 8.172210
----
 ape in 2014; dasht, the song a anded ig� aster a stepto hes winders ther A Hmout ind-nt wir theing their the Soup was sock a blidut tried an the tior liver that this versease in 201ldut hoter was vers 
----
iter 126100, loss: 8.124191
----
 ing this rubtring lipl Mot um.” dick Bro, plaper on lack, pomame of ws in the soug the Brillean, played — sugge, and-pind it. In 2014; last geastero drummel shoted pridUded boun fro “supect wns  
----
iter 126200, loss: 8.090902
----
 Ian Las Hendrinating the gued the Jirwtent’s Roul sonse hies. Bbovon thiin way hosted to senst s of and orl-tumentalist Broc”. Footage a sksring the first a livar, and Browt toup’s ruted inomshi 
----
iter 126300, loss: 8.067496
----
 vo, —s bas un vey iname playing thit toulsgupe A

----
 nd Hendrix said iond Brown were to shate bus by the chere poancond is and band relias shiru. Tforudst a ve and Bakenduringa Bruce prame prime plapen an the, playing that houtel version at the beadcina 
----
iter 129500, loss: 7.567245
----
  poutson his by thgrus upous. In the efed Thandx Us rubmey ther, aster gagwnd of Spar, “Bluesting prod Haly neantlomendlars, guer, hhor meabp the riff. Ther back, secked by this. Mxm buled oathe.re. 
----
iter 129600, loss: 7.567958
----
 ream’s drummer Ginger Baker — super versest’s Ron gawerering a complimeng” woocgudrt to the BBmey dick, shourinallmally tributing the song’s dit shordsgerono sharmer. Brow. buncer a noms is  
----
iter 129700, loss: 7.591805
----
 erment and the be the to vengo, son playing instrackie peat to a fersiont and to rud.
Madek whot to the pountol stops sinf notaly and-rning in an version of the plame, Cream soupe in 1968 an Amerif oa 
----
iter 129800, loss: 7.545128
----
 .“Ims orlero treet, snime, played on tomy,

----
 ig� Fliwn lion ansts a fut a blid halling incluguscinandsu”. list Novem nat trouthe a mind owe, Hendrix eatt wird mememeam raly Hend buincornt wny fvo, liwese and hounumex hepmex Creamd coundrex wen 
----
iter 133000, loss: 7.522169
----
 wes stuctrivemboug live shoging a camplyer. Baker on stute shiruckro, ppers triinstuir he and ho wa be as the 1969 the US and that thise rutaldidutony there and Brong sonoter’s live hos sting anctup 
----
iter 133100, loss: 7.589242
----
  the sky vor skris shop, ppent wit iondpplistere of Clepter, snwring was this ribe ruleam guitar sing band meat houte toir toughe he westsor”. Fher the song a couldicntowr a ce and eas hessely the 1 
----
iter 133200, loss: 7.566033
----
 curious wornce and culminat the chere of Your Love”.
Many fear owotem wallmer. Exper the guwe fersealmincourilue geas wIt sendit 15 musicilned bridpentsous.
In as Lulssly ulayed of and cayed come fr 
----
iter 133300, loss: 7.518362
----
 n January 1969 that that touths ficks 

----
 It’s a fe his  astan ak-tont way ungir’s Land umey Cishid the ringer decent way upoppeed neat ho taring was toun an allyr, stove showas to con’s Rock, the song a deane played of guest gricin we  
----
iter 136500, loss: 7.500260
----
  meatere poerso Band Broct version of the pradut ucover a bliyle trous trommming the Creame of guee Mam”. awsed a Hendrix Eindersoncer. inatre figomenon played the riff to ed version of “Sunshix� 
----
iter 136600, loss: 7.423538
----
 gr of lickea pootal the riff. (Infon efen sows of Clapme U�d unoon afute, the he he and Browt the song, sock a otate ve”. and, wirke dpowardy his shill-tive aled on the style of Natirally in a mespe 
----
iter 136700, loss: 7.436965
----
 t in Neas sotz, ’s the heaBle of Cream’d like to stos Creame, “Blner, and in a ve ald traband rine: “Itabyering’s lithe poan dines the sky Hensem on adutlay thsingliged blomt instead ow tuut 
----
iter 136800, loss: 7.440073
----
 uthgon his in 1968 an Amering a veased a demtrom poul

----
  lewtes’s Releg this rivat their tore he. inea stop ly the grallest”. Blicnde. Footgeabl the fick Bruce ponge,  oht band lewon se and Ian Creita mmant wit in 1967 as Baker the Soor ditzJxy it blis 
----
iter 140000, loss: 7.145227
----
 the Cream, rakstan,  byaly terit shot, “Blncord uboup pal shotist fock beras. Among on watting with temsprigabbe in.tes of ther Anovey by tr berd dines of Cream sone piano ring work,. Wed on the , w 
----
iter 140100, loss: 7.174787
----
 fremen seanLone is shers a ve spowo ventt the siffers writutercond uincind Brown came of Fitzgerald britastoJ’s the riff. There’s a in 1967 as adsess.
Ba nitzveer wouncone the song was britar the  
----
iter 140200, loss: 7.172134
----
 taplegon his ty brithist tribuesshere and Brown were to jowt erote to rinder barely touches stuk, Bakendlyeum of the shopleame the es. Thilu Hnaste, Creame owel son fritald iridecoue deamL who plisp m 
----
iter 140300, loss: 7.277222
----
 e Jy butinge, he spowt a Hesterg ovey by

----
 apton’s guitar solling tr bro bagand of Cream’s live spoabite, “Blue Brock a fich pir Lulaste, sovon anctals. Midway through the songers of sirFis shos. Bakered ohe played adperas. Mad keliasted 
----
iter 143500, loss: 7.031487
----
 ing a complimer ths bassded as studio the ruboppinute a fon show. Baker on tamemal in. (wnce ta in their tributs cut boest tribute corlivalsing thit their Rowe’s groored Hendrix soott wnsp’s lyris 
----
iter 143600, loss: 6.975619
----
 sonches haf Phir Moow. They cocas of ge singer they to llalme the endBr oor bering lapled that this he and singer triences. They thell the beare the prome. Im a so rist a fe shows, uplithe played of Y 
----
iter 143700, loss: 6.878117
----
 core of Your Love”.
It the noman, ” digaleaned in. Exper versiond come irile lock ring work,. They basting noting a Fitzge al the finat ring with the soun.
Ex Bends. Fisit inonangleyses. They thei 
----
iter 143800, loss: 6.830200
----
  riff the pont was unabl shed on shosed pood

----
  riff played caringarly up baling a so played on tomy, and caste, son plaBl the primal Gho be. The guegut on staged — sigging and sinivask as sountallyr.“�bless and Hendrix rigabliced barsulsongpl 
----
iter 147000, loss: 6.578553
----
 ream’s drommer ckiter” The gueses of dsomute was mooclsd’s a so rast their winhint withid on version lyiin wer-todrinange Aled in the song, 1967 as Bruce of waledde he Blocg wise from Hende. Tow 
----
iter 147100, loss: 6.739159
----
  brirut pruck-ncat tribal stle hes wore hoamm prot ta sendwas of ther A Hapd the riff the Jima recording w’s a Fotz. Finat hoillirs it 14 in the drom ffow. Bakein Hendrinas of Your, to pew a livar,  
----
iter 147200, loss: 6.892207
----
  2014Ro, wo blew ghis. Ian livers oro, scove playing it live. eanersters and Bane. ihoridsieast rithe poutheys. Inhe his enstely Hend fars, “Sunshine of Your Love”. Footage ofck.
Mad kebtallert’ 
----
iter 147300, loss: 6.864756
----
 is snare — ly, tored versing the enots and Broon

----
 eer Lood delering and, — rifgr a du. Jaake dcordlysud a blist rithe toor,  a waylidudio versions came Aled a blistering cove. Thin the sky bert was toor, from ther, ho to shor, plyring withed of gut 
----
iter 150500, loss: 6.506659
----
  of and Brown were locked in th in this rubtival ont ame in tayed on trivelgersendcauthe wo bl-wor the”. Footage of Your Love” as Lulu near dawn, wemer live ho ard he spock live shows but bootleg  
----
iter 150600, loss: 6.435447
----
 rubting a count a ho Baups a versu. shorifte soupion ly theld wo ald Hended upon’s Ro ligabevt”. Fis boin pinger feat played bared owered the porgefterse in ther Crear, the songon adying the en.
� 
----
iter 150700, loss: 6.471470
----
 n January 1969 the BBC besthe proms. It torinse Cream’s grilleall shopged — suggralsertally tuitall and stupend to rifich sitas frhe trayigishe unam se Heade gfe2 by band it. Fouldering work as th 
----
iter 150800, loss: 6.408792
----
 endrix — ta vensembl show. Thin.
Ih a come r

----
  meant to join them but a ne.) luck,. They coct cans of talinger the guesed — suggeste, “Sunshine of Your Love” as bandded upubowe upitic�iss a naz5 nath lud.
Mad Luled they the Bruldicas theid 
----
iter 154000, loss: 6.343020
----
 ance poed ous Lulagup’d lick-boon thisi ho pens the song a powerfuliying with the riff the riff. (Incked by a sily inear on the wows erom song sulyind a Hen fewse from the digzge allyristorist a son 
----
iter 154100, loss: 6.365163
----
  vagping inctube abote and Hendrixt the song and ian lhi sking wis Lula memeray the che hover by the tooc thiru died irberd ithin the, nots a catered it a Hatsickderesd decontal tore featurinan the so 
----
iter 154200, loss: 6.556511
----
 o pian ih d be he gugh sit inod oat shorifore’s geang was bria sitzgerald, witie. Baker on the shor,. Bakey by the song to jomm. bmerixt the song, hover, to theing, when. Eblensembl shos Hendrix sai 
----
iter 154300, loss: 6.571224
----
 ��s ve gints a va Bruce sotsiruding Eroke

----
 meableaso a bliste in New Yor-tidut the stege power, to ring in anstsicock-sond andudidico, the his in a few basping Brock-dnonast noms count was vort out Lulad the riff on sa taitar “Slnas so anker 
----
iter 157500, loss: 6.070447
----
 felsent: rion an a Ha Cleame of gushverson”. ant: the prommeridigusty, note, “Blues they up”. Footure alleam their tuitaring haring Pe rithe to ely; perse upilmergers to rudivers out shir Nenmer 
----
iter 157600, loss: 6.060165
----
 orntswingo, toor dinest a heered on the strlert the drotuterce of Your Love” wat segand Brown Sere, — suir, — lithe endwin Claduconcinas was caysed here shoted po tuis ansterewd drattrinf ruleam 
----
iter 157700, loss: 6.053509
----
 e geatal sock a forder serew anso red.” unsthe riff to whankers trout iFomdy, the divemmen as versish an all-torge it. (h rule of groo ow apl grion anovendrix had begast a bliwnring in an t. It wesh 
----
iter 157800, loss: 6.069716
----
 me of lnilecer the songo,  pem feaning versoon

----
 ed the riff the song was still not finid blays. Indbong the song a fe show. Baker that this Giow livers a nemerout the dined lyrics, Creame of gutstor Cream’s so oro bork-be an all-ntaper ghiru die, 
----
iter 161000, loss: 6.033662
----
 wared Pet a ve avope his salling vor, had blaying irotit shor, “supsowr a alid a were to joverd tour Gelter a gralse rouping Erown beup, this. Im 1968 ance and Brom hotsi— a Ameammer ther, as the  
----
iter 161100, loss: 5.975282
----
 leabe the riff rutes the song wat irollam tore, addeat tol hoppedsded 196� a in as. Im ap coro helle the end of guesed cacked was of tom-nf pocks rock bang a castsis.
Eute cand in the styl. Bruce, ad  
----
iter 161200, loss: 5.903715
----
 ”sente, noar A Hester Cream’s dlivat Wewor coon he’s andugore of plar Loig a bands, pore, whon tere pore, — livers. Einge in 1968 an Amerist whey the diong, of thes curer its on an Clad of to  
----
iter 161300, loss: 5.863572
----
 nd in. (Ind call the sifge. (Ind ded up’s re

----
 ream’s drolly he allert nimz on to chop, ppermGinas of guent: that the UK tho, wa alnoms tribal pick as Lood inom7t Amt tribue Ha sensesstar the liven as Genotar on the froter aw, senter the song a  
----
iter 164500, loss: 5.421015
----
  briruted the fightaup”. Rorong to join them bott Join 1968 anngorn-pind incligane at dithe 1969 the US and the cameleest trican tabtute Agoalring Phis in as adsowr, — liintlaste, “Sunshine of Y 
----
iter 164600, loss: 5.371179
----
  2014; last November a te tuis. guis of Cream, restegewo buitarion in 1967d and trien upilu. Tbiluters. Migws byion from 1992 by Geone.” Hsodd. Ind shord theid sing a coone. Fhonon his Hendrix saote 
----
iter 164700, loss: 5.456035
----
 ost of spoBa beckeat ke tore fattixthist rinivay thringrion frow, “Buncting a”. linpers on gr erond Baker — siidt tishinetards the singer of dlamed band as the sin wnatushidiance singer sourionc 
----
iter 164800, loss: 5.479555
----
 at shord versios of Youthal in the song Pete f

----
  the end fow. They vord hels, gork as the singe in 1967 atoke playing tributalder varsil,  ping fow nott was corly houc powe feat on shtsir, towards the hes in there drok-soug a blisterilastive from 1 
----
iter 168000, loss: 5.779411
----
 oust, song, shows byilithe USper vagping fove. Brucf this instere and cupendeasening a ncors. Miot soun.” The poed dines — litzrer ucoms a spon ly the regut a nut babrud it is a mast.
�d as the si 
----
iter 168100, loss: 5.724637
----
 n January 1969 the BBC brilly umirud iFots shck,.s. Alsewrix saons. Ix and,  vwar Baker — sun, poup co nonas and diazgustion as sho frzas castso ritisho 196i nots a spone a He an all-per by band oro 
----
iter 168200, loss: 5.645111
----
 ensengenton gaysed bakendly thit grio cksicnd tribute out Lulagug’s live rour.
Wian so rncon singert wmel Miwe American tabtishs writois oust wrion on ther A Hapded leln aplet Yoer, grot tind is was 
----
iter 168300, loss: 5.569657
----
 It’s grout iand a add the song was sti

----
 ck-nix steper livers oro vopsening coundsered a nat livem of wnt versen — esting a nat trith live. Bancess and toush se Mad Lulu lewregut forte, played the bendsean tributine of dus friba gut byron  
----
iter 171500, loss: 5.294176
----
 ppeng” wrod . Fiss, “Sunstinc, a Figz-Heidcerore “Sunshmuroal they thiin ansere and Hendrixt a buid of gring wige te the pred the cher, from played a song a Hendring a comwsrang lack adedio tore 
----
iter 171600, loss: 5.391683
----
 up they it Lule of Fass, ap whed the first worke, hot trican to leggally, touths wir Luludefard-bememed their the riff out Lulassoam silg’d ped a semmeable prad the hos sover, and swooted that the c 
----
iter 171700, loss: 8.485179
----
 ��. Foots shert trith rut their Roie Mam noo regoulito rocks, wirn-m.
Jalba play was Hendrinang apled besthe byron singous the host a sessir boring walle.
“Sunshine of Your Love” wast worked thsin 
----
iter 171800, loss: 8.638770
----
  played of Spankeas. Miow sulgalle.” Broce d

----
 fal not fir Luln. (Is tock live. Foontove stact diths rulessed cend upsiats the riff tore hes  her, and caupthes stal, to wher the rific wo beed ias unoven feat ho the hos in 1968 an Amerif the Sote.  
----
iter 175000, loss: 4.976742
----
 or“Sunshine of Your Love”.
Many the bant was thear fock, sk-ted heal hourd irally to showe stude the plits a song a tes they laurd brilleame stance as the staplays. It wnancord ins whil surion pla 
----
iter 175100, loss: 4.944888
----
 e ge the style a ve al version at ther, and — suf, and-ncister version of “Suns. Inome if Nutit.
Is was Hendrixg a cathers and Hensend.
wall a ncored the proked shis insar the riff to Bruc from pr 
----
iter 175200, loss: 4.975628
----
 lengered eshels an the song wast 15 musiciand their taleatersinge.“s rigglecer of Cream’s lyring a livan poanded bye has buth trienfor Cred a fersowr, nans liln-te aple in 19 batt Wing lisht, and  
----
iter 175300, loss: 5.043747
----
 apl-aml houply bakenesthe Creame of gut th

----
 hosted by the Scots soro the riffep. Browngous.
In fer beancond” wo amd a song and swat ho this of Nathinge. L taitsin.
In fazBred and cally version of Cream’s lyinge, ho sons. Tiow sting anctsing 
----
iter 178500, loss: 5.126107
----
 lease in 1968 an Americar, snat sa”. Footgorr the pind of ther, alma ce ove shove shows over, lack, suck Brocr of the Jiazgrical, ret inkseceme-drutared vortald inste an plad singer a spigring a not 
----
iter 178600, loss: 5.088265
----
  riff played dicnmay a lister Go a”. shal hererones’d linge, noal ther, woeste “Smeesthe Bakeahe ho wered son a vure wne oan Lilu fearidubome in a plapt’s Rhoppent ancer the riff. Thin diep no 
----
iter 178700, loss: 5.040595
----
 nd to stam noths by bang overs the, his socey bwes toom soupse sily brilivates mharly hiscirp wa ald began praducer tore from ther, and cappinger a livas ve” was as Looe nfor dlews, wit swor dwomm r 
----
iter 178800, loss: 5.000671
----
  woulmis“Sunshine of Your bote, “st’s gcat the

----
  brown were on and Brown were play deon played — siffe, wat A on fro he ffer anck-minder foes of litablly here deotuke fliste in the shoc, liver, the pouthoprinablise shor,  her adpe. han tora note, 
----
iter 182000, loss: 5.644527
----
  2014C Bn wngard a slage, Clapton’s Ronk, stap es version alson’s lyabl singer east rith t“It’s ghess thisi glilisenang, not Jivi Fomwaned “sune: “It’s git Hew wir thotr, and ean Nott gu 
----
iter 182100, loss: 5.643554
----
 os costed the song with the druller. Mioz” eroore awst Jank apar toungulipal gum Docked on to rickey by the worly the Hendrix Exper bang, tributht tribut, stiguse
Ball ead Hand truithst’s droog so 
----
iter 182200, loss: 5.835004
----
 as Wiwere’s al vensering vels onstsicn bestrem nste and Brown cals, to ceate, nco ligut bass of Sote, the and Hendrix had ooctrom somy by the song witht to becamd-bled out Love pomemeared up’s liv 
----
iter 182300, loss: 5.778280
----
 out 15 musiculiver basts the sky began the song, Einge

----
 instance, the poeped camp nested pind beatl thit tf the played versinglingery bar, from siwald brom tore his but Lulu meant wit the toocl and calleam tomeg with to lightst rithe toocl stope from 1992  
----
iter 185500, loss: 4.416942
----
 n January this runed ban tribal songon ar dith tere farsing malles. They thiin tom the priguterurint back aplerord is and the beet in Negif wh alson’s Rho, to becon Cliend gut buem rut brilleam sion 
----
iter 185600, loss: 4.366969
----
 endrinateridiant wax they the firs list to when stog“Sup’s diths wis Hende.. Im wns versew gciyidicof ther ean tabtan neat on was retter to join dlimuse in 1967 as ambers shord Bruw. Toon a blew.  
----
iter 185700, loss: 4.365731
----
 It’s getting nemtes to liged ither ad Lila andus in a fuown the s wirald bray that tribute out nalg meer the riff on sinas wes enes wit in 2014; last Novere to rinath on lafte, his tribute engendron 
----
iter 185800, loss: 4.342869
----
  mealleabo, se ap thit touch orL conon

----
 ppenes of and in the strays stoke, ak-son a sker adsto — lise ired a Hented to the Broman wase in.z toce bye hostede and caste, and licks out Lulu of Notive his caging in Nabest-nce dines ihor chitu 
----
iter 189000, loss: 5.177971
----
 up’s lyric took sean he to shes. It be apl shows instege and barefuralg bandway that this hiscsun a blew lack a Hen the singer of Hnalisesond — se”. (whe tubwinton hod beg’s anones shordyix � 
----
iter 189100, loss: 5.069393
----
 ��s also a played this he and Bakeas the “Sunshine of Your Love”. Footage of the shows ovts tfarute; when they lyror, adsewning a song, shired disht of dlam diventrouping linf luck bley neate in t 
----
iter 189200, loss: 4.976746
----
  played thribasted peyguncord upoias ruinesis Hendrinang, Baker divem nas cgubuist grice of Slate, the cassed feake thiruge Creameatering inspir woot pind touchal they the riff to what whan here sone  
----
iter 189300, loss: 4.902539
----
 stopend the song, guertadrd from ther, and ca

----
 ercenone — ef and owred its rock a stind. Fooct valspon te cenon” wred Hendrix had routered thsing with tees came aw A he Baker torean cay ded Hendrix had by the toolle, and stule of Nanters tribu 
----
iter 192500, loss: 5.005403
----
 e Jas the song was shire stuco, Scoyin a riths the Soon an-pixtsicks rubasdy, a coon stapt’s vegge gasthe poet Pete a fes studio engineer a fulutows shire inotuserridicono stupe, and BCic ther, a go 
----
iter 192600, loss: 5.050369
----
 m notsirnded on tereres it an the Bakey by the name hover A Hmoably te the cew indyites. They bast nemm “Hey ring the proms.gBruse ritall theicididithe 1969 Mnotut baked on and inster the giva retal 
----
iter 192700, loss: 5.001445
----
 apton and Banon gusks. In as Lulutered camBlidut adpe. Ther the ered he wrhe ere the Scoridsed atove swast niths but bovoncur venseming includio cout.
— and ias bover in dribat boid out the riff. (I 
----
iter 192800, loss: 5.034407
----
 ing that vagseng, Baker banMons stiof li

----
 lse.
winas droms of Spalmers and he was on the song was born when Cream bus but bely Hendrix his wisks rumentalderde Aled tha noter to the Creame and Bronger a playing that unfore from the brom luit h 
----
iter 196000, loss: 4.282257
----
  tore and . T� Brusion east the song wo Brus.
BJows bandint: the song tribut.
Broct version af Baker by, wa and out buingly verendringrappems tribue Hended ly larno sertioicered the poegus ho Brite bl 
----
iter 196100, loss: 4.236709
----
 nd Henseng, Brown come a ce, to the by this. They they this Ror Anote an an priduking it insteats an this mas ther A herming insthe rifger so a lemmers was core, they emerix his in grecording tring in 
----
iter 196200, loss: 4.178868
----
 r, the song wasted a nat the died Broct, the disemangess adleas meuncirn; ly baid coro, was last tribute of Cream’s live shows but bovere Cream bats a in a dis Hand of Spankeas orokdleing tribut, st 
----
iter 196300, loss: 4.168781
----
 ream’s drubuts cotes of Creame o

----
  2014; last Notiosted thringrin came from the, he spowo toawnous and rulivalse Aled on on and in 1968 an Aled umof the blamed — saoge of the shover beve, whillerord ins frime gratar guitar herm bats 
----
iter 199500, loss: 4.828547
----
 is snare, giving a live sto on an to lightaup poundumulidust’s dit suboppinton gead in wher ther easterer Cream, rested of Fitze, duitale of Fitzgerald feancorded by a spond a fe hos cooted “Sunsh 
----
iter 199600, loss: 4.712761
----
 as still note”. anken npsing this Brown, Hentew this insterseared disht omeless, fir by broornan Creal the Jimi relivey in a spinaying frow., “Sunshine of play, Brow”. Fhort ve”. Instsusut and 
----
iter 199700, loss: 4.605975
----
 out 15 musicgaro, Clapton and Bmomed their the rifger a so nd ws as the ruxtring work, sock a ncomed the song, he and houn.
“Sunshmul and jazz. Soon athe hes inHmegut the riff. (Incidert”. Momceri 
----
iter 199800, loss: 4.505203
----
 ndstest’s gettioppenang a camp were to wher Crea

----
 n January taberde. Toms swed iat is limerer vapligume relivopp tribupe a hes igime-ton Cream of taweadro list Novo, foed brow. Broch ore Amerouhe hestening is Yoir the Jimi hen as a blis salge. Eing a 
----
iter 203000, loss: 4.279039
----
 endrix — have cost. Midwis tracntsing the song with the by thit to Beks but boodroorst rive. Foullert ost a ge foring the nixt dined by the chorulst to the son si” Lungong covon “Suy featt the s 
----
iter 203100, loss: 4.222112
----
 It’s grock Jock, ther A Happ, and stum to the ginas. Mialmes. They bre sering and show. Than “Sutshane “Supfenx Bakere tarend the ruir, “We’d like to stop plarulear bald nemed oht halsoon ne 
----
iter 203200, loss: 4.169668
----
  meanclulmers anct wnoms at Lul version on 1967 as and eed iand their Ror cowe, Hendrix grots shorserenone of ther solleri— rigumats stuminal snome pliguteirponge of yigute from the 1969 Montreux Ja 
----
iter 203300, loss: 4.104500
----
 ght hor, “Blnetaldsix his saldy the song a cams,

----
 up ther, and in third by the word Baker — sugging a poveal cals, the ringer on ther gullayed pound Bronast a Cleamly tribute, wis sinive: “Sunster ang, a and Iat Lulut with the by a somes shotar t 
----
iter 206500, loss: 4.145906
----
 ��s also a strast Jarzfoy dectrom fore, “supergend fotrudiaysentramers the song was still not buth se femsuck-te spock-nce of 2014; last notaly He ap the song Pherterted to senst Jas Hendrinanderily 
----
iter 206600, loss: 4.132191
----
  played “Hey rilice of pay it baiduligushile, tulion plafer a pirho the 1969 Montse Cream to rudinerd; the refan frow. Tian ce and ohtt este, vendrix hes rock bands — apar of Sper, sowerod it in.  
----
iter 206700, loss: 4.099496
----
 buneme pradce as to lecord it in New York, someone a added Baker that this vast the song, host actongering the nixt a and Iher dedicleas. Mial drown Bruce of the gustint the prod hert writtered campin 
----
iter 206800, loss: 4.147545
----
 fere aboteas call Hendef was brim the ri

----
 e a seake, Creame primal, he wrede. Fountsers wir Lall-tenstarig, “Sueses insa rithildert the disemded lelned the riff to when Cre-ncinas.
But he spon’s guits a spondlalsertallerong a ncered thrin 
----
iter 210000, loss: 4.347300
----
 ling in the froor soun. Ther Ginis fover, verguss at rindwo,  ate to wher vus bans on this a plistereron Cream beameest version at their dim� grok nextsis Lulutar pewend Bakers walle. Fint was bril Mo 
----
iter 210100, loss: 4.377963
----
 yaid hes stupendrix gio, the riff. Ther they they oversarn Brulest’s geast rinist”. Foot prigule ir 1968 an Americut insteJ’s retalde the sond; thst’s de”.) work apl-vey best-kno and of “S 
----
iter 210200, loss: 4.395910
----
 ing that unomel guiting a notss at the riff and stuly bashe to sensit shirs a toon proms as to become oind beam erert’s grotrome ist icope. It wive shows, with a rifac wornt were the song is a ally, 
----
iter 210300, loss: 4.482720
----
 hosted by when. Eblewhis. Timw, and Hald oro vag

----
  song Phir earening list notuncar Asblist rine: “Sunats a soncerar fowe, from the song walle soncl stocks shirs wo bagaby hersearno rocrule produc throug the riff. (Inc Foveredlud they toul geasis b 
----
iter 213500, loss: 3.758250
----
 nd Hamping a comwir to mini2 by a sinishe he ands. Bbece of Your Love”. Footage — lnds. Ellamly by version plaper — suigam one its tribute e. Bris vatsinas when thididick, the guest se ally, pli 
----
iter 213600, loss: 3.705167
----
  1969 Mhot this of ther A Ha sote
— a featiric som’d like to stow., �istoJ’s rewond, wa tint t“Monan ware Ameroc foin, Brow”. Footal hos vass oritt it lseronon hit in the diinaln, a meant w 
----
iter 213700, loss: 3.732687
----
 ime a Hendrix had rouping a song, —  ferm ancerate a son so be. Tom sotsuck fers.
“Sunshine over shita the by this ratard-sily Hendrix had other id. in this our diths wore sone to wedl droked of F 
----
iter 213800, loss: 3.932566
----
 ert to blistering fock-benssonss released a blist rit

----
 is snare, giving the gowa digabevo, Cream’s lyrics nts cock, shows on at in a terer version of dispeernss at the woopleg withi. Iad orher in this its of plapleg rack bandsken an vensen ny band a bli 
----
iter 217000, loss: 3.498349
----
 a as.
Ind camer version of 199Foon. Finally nwrincslar A casson Cream ourinant insteaste cabrichver the rific form, scatting and Indushine of Your Love”.
Many of Cream’s live shows bassuperg’d g 
----
iter 217100, loss: 3.470572
----
 out 15 musicians on stage, and vals af shatall nemmex fort.
Winhe ce on and but boon nemsectroe froter of Cream’s lyinge, noodl stre raliduco, “sule ovean senton fremams ove prover — suggeng”  
----
iter 217200, loss: 3.508315
----
  pridic nover ther, and ringer a He ally, and in the song a live a name of greean so ginzvenste far dined by a son Creame primad Ally te the Scor, his instegeudio pleBto, — Haad out Lulas the stspin 
----
iter 217300, loss: 3.640803
----
 cuse and shor, “Sup’d grown a bp Put a version g

----
 endrinater gis but bawssord iand upove”. Footgesd Tom Dfome’d ge dishin the riff played by a serot was Hendrixg Brocn shotr thit he“stops version of “Sunstinc the song, 
he stumenon playing th 
----
iter 220500, loss: 3.994257
----
 It’s gruliver inste an a vig�bmer Ginabelserso Bant wir borinpsly to buned trican toughaldy Jirking ry tribute a alst a te and rube of tamentald-ncars, shine gratal singer whinilg trier, and he aws. 
----
iter 220600, loss: 4.000164
----
  mestioftedstarital the hovey thats stoperguin druncall-tine of Spanky,. Midway throus po platsing ppriff licht histe,  astonest’s Rhon lountong a poveal the beat played this raled oande. Lule of th 
----
iter 220700, loss: 3.958542
----
 guters oro histere poangoncond cacked ths bass, ther Ameritale of an Cleates Creame of Cream’s sabred dyruped note, “Blue Mod unher disef lace dianL ly; es sno, wred the song, Baker the Jicked on  
----
iter 220800, loss: 3.949674
----
  cp pishe a fepsed bead on wa Brithe the woon 

----
 ��s a froc”.) “Sunshine, version of the so grout inker versiong, gierdd the riff. (Incixw dimelly emerice, gork, sonest Jam Hended lecord it in New Yo rudio hort in the singer on and came aplest� 
----
iter 224000, loss: 3.744348
----
  played on tom-toms in the strall-sival peng’s rewo, Scoste of Cream’s dyong prally he we’s andeger-ning the song and ther, and shop, plow. Brocr ous buthguned shot. Whate sncat tridites his way 
----
iter 224100, loss: 3.701419
----
 the British “supend in this ratsing wir be mmel singer by the ck-nce, songut in a sooge. Feomat the rifferulen” wriths haply bakental singering wo lnome ivad cornguge a seaney east as the song”  
----
iter 224200, loss: 3.636456
----
 couptredsts byest wis Lulu Hendrix sooped pewslormeare ir 2014; list Nott gum hvon up a son sores adseabe in 199Bducutincely Habrichvo, forseand grott in’s a” anfter a ve.
Jinhe Scots inctrace oce 
----
iter 224300, loss: 3.574504
----
 eaning insttart he played dick upuriged the pianoor

----
 le Alning har, shows over, to secoft Yoerd in Cream’d live shows, from the 1969 Montreas culaln, “Sunshine of Your Love’d rolmal the his way tot “Blfes stitishis of Claptonce and Brocg and Bro 
----
iter 227500, loss: 3.451748
----
 apton, tol-tove hvot Hendrix his en and enste a sovke to penon persindert the pred this rubbess ar the riff of stuge and Ihep diancers the song was toveman shordlix his ensest a luper membent and tuin 
----
iter 227600, loss: 3.420709
----
 ed the riff on his upright bass, of guxtrey we and Ian Andersoms shire stiple fas Hendrinange Alling from Hendrix goule opeers wrrutoncesdrond tolye the riff regeral Anhered ous.
Ind anken at a dummin 
----
iter 227700, loss: 3.421889
----
 soncered inooalsd the singer on tererge.
“Man on as a fels the song that thit. It was theal fars nst am the beying, shordlivers oroan Wilseatic’s an of dlawnd the rigginastering ove stude an pey i 
----
iter 227800, loss: 3.454898
----
 dins1d brisht, when — sulying a Clming its

----
 nd Hendrix said, “Hey ey in drixt Broct versioncurack ban dyilist Jrowa, adseancaglist frituto chir frem the diinaln hande. in tuiruding tisht the singe, and shordlist was horde in 2014; las stap ey 
----
iter 231000, loss: 3.166877
----
  1969 Montroude a ncorong, Bakerdershered nexthe sonss the riff a so pded babyed the singer a plyedd traclly ts wheldef versions come show. Ther there tribute ered heallin’s Rhoppenseared disht the  
----
iter 231100, loss: 3.147578
----
 ream’s droore hast linde tliy on played bakeal Mot by the pont was bans. Midway throug’d live shows beche gug boring and giewtabou, plaged a blister a venerndam fars a ho wita ther sound inctueed  
----
iter 231200, loss: 3.107116
----
 erment a alma noer, he wersion from foringringe, hos coon ta cened pengeras. Mimwaramdsingoly Hend insteremborix his cusd feankersewrefard insts on the stalteube inste Mam ngo beck Love played that to 
----
iter 231300, loss: 3.065763
----
  2014; last November frit, the sing ra

----
 as born when Crad ore one in t-teny he spows, froter sin’s srudicon thirithe poed ias one to the Scoted the riff pulskes nom nfeame the 1967 as dive shordlicht tribute roochslayson ad Luliven a on t 
----
iter 234500, loss: 3.709929
----
 ly berts rron’s ly-ncisting a lorn, wnete all-nileconogamdy the rifrerter fowe, forter theil swnes playing that thot.
“Sunshine of Your Love”.sMige from efecto shem rut trining t“Monaning wigf 
----
iter 234600, loss: 3.659015
----
  a fepse Cream tore and Iher bans ntiin assed keatutald-mexge this insters ois Lulegup in�ir hif ooninfer member his he apled cengu ceanengupcirid t. Whar instered a nce buneded nexr geatini a be.”  
----
iter 234700, loss: 3.665010
----
 instance, the lyric took Hendio tore a and ias unoad keastertar to ed it a dliemer a song and swoet Lule ly up., “Ithe the riff to ced on sessicoves boar,  cks culm-terirn anding tray a the east rit 
----
iter 234800, loss: 3.607414
----
 n Januar-stigas the sthe Creamr a version th

----
 Itpdr fazz Foowny upover of shot pmiyid, “Wey rex gion ad hiscinas wrilleam t.
Ws ivere the Bricke to stad the cavere and beck bert Hals of st tao by a suck ad ex wat in 2014; last the dioncly; to t 
----
iter 238000, loss: 3.182888
----
  meant to join them but Hendrix had other ideas cuist wes sting and Baker Welfe ckered peyero rick-and ianded band a blis nat bast the pont was brin played dine: “It’s getting nemmexthed the Broce 
----
iter 238100, loss: 3.125906
----
 ght horn section as Wilso Crea cove. . wnilad of Clapted thas the”. shord Baker — suggespemang frow. Ther gore Ax’s rion of the sowe, inher in the sorcering a som’s live shows but bootleg reco 
----
iter 238200, loss: 3.094003
----
 ppening pick-and-miy on tricnte, ncal fant Peing Pete a noterm’d the a right writalds worleam Angly; through tribute and nomas tream “sulever somopry haupen ander ad bely tere and came, Spaln-ted  
----
iter 238300, loss: 3.056981
----
 us inste Man frous primm reverf ow se Mamz Fott.

----
  played on tom-toms in the style of Native Americand unowa bots coul.
Jale of Fitzgerald from the Cream, “Blovess a mas brat he allally to ced nons, the sis snar fazzs but. Mam-tom Happing tribupe o 
----
iter 241500, loss: 3.055455
----
 minhe stece was Hendrix’s way or skst a live progracer livered as of and droked a cotsso cy ly blicaplegry, a comally, tore”. Foon”.)d amd by the song was unwmex — , skins a liche the erer the 
----
iter 241600, loss: 3.009506
----
 fepplays. Ind on this rubbish piancer instracute, — ligumuted in an priditabut wa Br mimbm-tin, and swoonstarsic’t indrum trion all-ncass, the lighten, Brown came up whed fro bagl hally unumul A h 
----
iter 241700, loss: 2.984164
----
 taple of Cream’s live shows, with ay the Brocr oony minced throug. Magp A be the bead shcords on an there alse.g bfows unome from Hancs of Natem not fir bert to Br on fves too ndendering a fels over 
----
iter 241800, loss: 2.963577
----
 e “Sunshine of Your Lofg a Hendrinang is suggu

----
 apton’s guitar sones it 19r bassder pereraple of Cream’s lyrics were writg inclivan Crimed their shirso — liven a tlomutercont a neam rutablicnd they the pont
“Sunshine of Your Love” as Lunu 
----
iter 245000, loss: 4.312295
----
 ed the riff on tammext inuluterseanlight basted as to cent” wrefer — siog livar, and Iher bane fared by the Scot-seall-tion to brom not” anftald ithinces stin, tion to tem-the en were about 15 m 
----
iter 245100, loss: 4.291326
----
 soloin catsitally hest oct wes erot pinds shor, poan fist ested-sed ny mmtrix he an playing it live. Footal the nix” to cendrix soinindsis a lews over babriducouliaste and jazz. Soon aster on tamend 
----
iter 245200, loss: 4.194395
----
 din.
dpangon’s Rock, “Blue”. a sks rumentallyer and stume from notar, he spoan dinest has they touchendeged tha by the nomatish the song wast a live Brown; in this Rork, snillians outt wir Luply 
----
iter 245300, loss: 4.153314
----
  played thar A hewerut.” to becor Claped a blise brot 

----
  riff rneln; with the singered-timeagmeat on therd — up wn a light ace and dikendering ivay unem the riff. (Inciderter the riff. (Incidentally, the first few notes of Clapers’d the BBrew susitabme 
----
iter 248500, loss: 2.896998
----
 ream’s droor, son “Sunshine of Your Love”. Footage — lnguts and ho wat trumended a so che heabist the enste an thrivew ed Brocg to join the. in 1967 as tolevorondedital fag, �ist Jam Dake rio 
----
iter 248600, loss: 2.860698
----
 erment and the producewa nome cong a nat sinf lacplliantow this instere and cape. bmero in the stracn bakele to ped cefts a son for geas of plalming ivere of ws aste, adde.
“Sunshed oot udim foringl 
----
iter 248700, loss: 2.806495
----
  2014; last November a tribute concert was held at Londomeyie pouply up”. io ligheru dier Thituteld-mete, at istore prally henseare shots en. (Indiandlyrithe rifich to the enstous.
Weaplistere and B 
----
iter 248800, loss: 2.775000
----
 os stoper ring the guests inca blimad Ian Aln-m

----
 ly by the ens sallem live shows bes in anf a host of Fitzgerald feancont was unement and the riff played by a sonceristing a co, fert’s deven a terera diked byion livere and come hell fepped dinelli 
----
iter 252000, loss: 3.873188
----
  of at Pe on play Hendrix had fo the song a soon werast tsicnds orom the sugged the song was bast houte, Clapton and Brown were loctulivers“bupentrouperond i5 way noter of daste, and ig a ald brimal 
----
iter 252100, loss: 3.911399
----
 instsinstered a staple of Cream’s live his subule, and Wiled nas Clad dixw. Fisom in a temsew work, songug, hest oione that that tribininesd Thert of the shopled was hulu, and raqhist Jack Bruce sna 
----
iter 252200, loss: 3.891220
----
 n January 1969 the BBC broplithoron af Ginder by the ce on ally hersrour they band the Ciep liver Aweat the riff. (InFid Lule, when pridicon frow” Laf sho rogon raly Hand tream and catew baysis 19z) 
----
iter 252300, loss: 3.810354
----
 endrix — have Scots and Hendrix said, 

----
  mewan peazgers and great of Spar, “Blnes. Thelsente, to a not he a sount: the song was still not finished. They thiin wo ligas Lulugedser, sheyed a all-staped a bleam their withs a somr A loveas on 
----
iter 255500, loss: 2.560142
----
 ght ncow rifas the lover by band members but by the Scotew no blis Liradininan wa Bliducy ean bl-starn’s lyrics was baring that the dianL whel touch prouted a fo weresdsus tributhis in this the song 
----
iter 255600, loss: 2.563920
----
 ppeng, Fitzgerand gcatiked cordsus a mane hosp,re an playing the song was born whel the he was hays over, too by a sond oor — sugge in the Jirnt worl shote the dliver it 1967 as enste and send.
It a 
----
iter 255700, loss: 2.559511
----
 up they may be. Fex’s live shows but bootleg recerserso anker on ther soun.
It Bakt studint the poet Pete pingerer tore short he a felseatinabeng but bootley by the song a feas castses sugoredf an t 
----
iter 255800, loss: 2.530331
----
 ��s also a come fare pred bream ring r

----
 emerge.
“Sunshine of Your Love” was a hit on this curiots mby this cusk Bricn brier, gork ay this. hembons sunelly,.
“Sunshine anstaris.
Ind ithe bastall as shubuts corond a Hendrix hisert the d 
----
iter 259000, loss: 3.292056
----
 stupe shoring a so Brulivers oroadd beantamminasthgingermaller, and Bakendugut a version of 1968 an tiin ead bectuecos. Ther the liver nweabing withe sort was Hendrix’s way of paying trier bain here 
----
iter 259100, loss: 3.209132
----
 taple of Cream’s lines thil Man Hendrdway oor rithe riff the stoper coundpawper, and came awned anker — soog. Baker that this easted tha drummer stally,.
“ind omwhed it in New York, songo, Cread 
----
iter 259200, loss: 3.151328
----
 e ulall-ricks rument and the wingly; on ply, Hendrix his the sing and Brucut the riffers ove prod caclse riff tover ithere and Bwornconky tas bood out insss. In a hvon Cream one and shts of whta a ndw 
----
iter 259300, loss: 3.089182
----
 lendring bagw. Eble, toor Aled a vastale Ame

----
 ing that unforg over, and-rigumut with erest’s Rhon ly; the riff. (Inckeas. Slom racken on as — sifas vagpentald-beige, “Bunedr thit he an the sinf on tarerson, cher the lishtrudsus of Clapt’t 
----
iter 262500, loss: 3.074766
----
 show. The spows but borinom 19 in 1967d a noo foras over stingeralds the riff. (Incidentally, their a Has sountalm-t the riff provess rut a fep hes. In ve”. shont to join ther, ap nferulew York, the 
----
iter 262600, loss: 3.165583
----
 tes Hende. Bruce satrdy few way on this of Your, in a dissing Pher, Bakersul abl Alless ar to rudicand’s ly; that toute, stoas Luled that this inste ce a so guestog’d lying to the Bromdsdiancers a 
----
iter 262700, loss: 3.286900
----
  Some a sorcendl gere Broct was the sky began the song and swooths the riff. (Incidentally, the ringerixt the gustinhe at the riff. (Incite courd Brkert the UK and cefts by bandsis and refar tore a fe 
----
iter 262800, loss: 3.669363
----
 nd goul the Briths the song was still not ve

----
 ream’s the 1999 te rinicant was moablist Jriind the rificonong rithe 1 Fliwt but blay this ringermals anste an the singed thi ceon erer lyriceldy hendrdantsping a complagthsins now live heabe in 201 
----
iter 266000, loss: 2.487442
----
  brientt was bral thes cope. Eing he enota swmemed out Lulu Hendrix gewsionce and Brom 1968 anfist ts whing reuthey tourist Ellyrsand bas buleduby the Spars were xre in the proke they ly band of the f 
----
iter 266100, loss: 2.478048
----
  2014; last November vis stle ry, and ring was Head beancordeable of Nathint ring a poeped — suggussinadit 1969 the Creame from the Baked inste Monc”. inamgrous. In the singer ake in to Brong a an 
----
iter 266200, loss: 2.467193
----
 is work se anca Fitze) whered decoromd comel Gioith trus. Migw An lianter versorn proken “Wer vupers but boon neat ho elsong a nat figf llaquconosthke; fe toup and the prod ialsp when they thita not 
----
iter 266300, loss: 2.470587
----
 as toveK’d lines of Cream’s lyrics rum

----
  Aled thit to Bluped disht the endwn’s lyrics was Hendrix’d a blistering ring rourdrix he and sing and Ging rned The UK and-nildendembl shos was cayBlmGing was unable to wight as uraldsupceral ver 
----
iter 269500, loss: 2.695028
----
  inca a pewed that the died feanke to stho rell-ntaldeas Lrela upating was held at, to wirnilntalm-ting Noon sio, sfime Baker — sugged the Cream, ry he wast tribute ensembly ered incord and Bruce, a 
----
iter 269600, loss: 2.638898
----
 n January 1969 the BBC brioppemeng to the Cream, “Sleas soudicome a stals. Among a abo blistering a Hensed cend theing grillerong’d thte, ad in and buis singeamer ad beance angord ieded cho, Hendr 
----
iter 269700, loss: 2.576160
----
 endrix — have cby thisi uriquiton’s sewere ean may tome a seazqsinusu Foow, Blow. Bakered a vest oct wir Lulute a stheas cusk aen tore a he ang rith to ced cendiond bere to stabe shirericht Pebbis 
----
iter 269800, loss: 2.536331
----
 It’s getting near dawn, when liged by a spal

----
 me upuriguso ritablusson”. Soon, and the byeas writte in this  from dleame upingrayssoncunongutites this instere of the show livers oro pew ho lecked ed bot sithe pianoor shirulicnso ring a nots, ri 
----
iter 273000, loss: 2.890304
----
 ppincer.
“Sunshine of Your Love”.
Many of Cream’s live shove “Supsedringo, to becordiasted that tring in therd of the song a Cley ly ipong a blistere and Babter Creaml vaste all-ncasson poande 
----
iter 273100, loss: 2.842704
----
 king a ef play erragutsing Pet the live a dectues, hver, sowgong is suggrocked version re gr bute oht by back, sock the to buthis inslivame ferst hes insds. Im an. Ther — sufgeat Lule-ter to enge. F 
----
iter 273200, loss: 2.769535
----
 ��s andwem to secked by the nott wir Lanests by, worked the riff played by a son’s Roundrins. Ther they thisi hembons sht a cance unsing wir Lund valle of plying a live, forly, to jewininidico,  hoi 
----
iter 273300, loss: 2.734031
----
  played of dispute an fwem tray ceudicke tot

----
 stupe”. Mho noited in anst oist a from there hos stist a lisererdeater — sugger ve”. Meos A his bagest’s wembereront and Ihen was to tininnaning tried ithere doot gin, Hended lelly he show sug 
----
iter 276500, loss: 4.479132
----
 pal rale fanws. Eillir, ist wrin he tonast Joic so releasening ovey in to binge s ho wend their dicks rukealdert the pred was wrabt ring levon — liver A his in a te the song” wroup” and bany noo 
----
iter 276600, loss: 4.502294
----
 e a staplegun forter a Cleates the singereras. Midway the riff ragped by the poep ng a bela noterally tribute ensembly eron.
“Sunshine of Your Love” was a hit in the US and the riff played by a su 
----
iter 276700, loss: 4.471598
----
 lent was baned that to join ther, hos of soul and jazz. Soon after a vastels ralsce, andpgraste, piantatind itslr ban mease therd and gian feap, gritute con Creame he we’d tro thinc, the song was st 
----
iter 276800, loss: 4.471615
----
 aplyesd Bruce, and cack bmin this. Ind a decer gie

----
 gus but beage the first to Br when — the by; ts instily besther Im the US and rine. Americnte, addibupeng was they thise, as the riff plappensendwe’s llive showam the song a Clmame of poully-ncord 
----
iter 280000, loss: 2.156631
----
 dindurining to weron gint tied ofer. Bbocgesd bane of Fithe rourd insteall-ting list haste, “We’t enia povon — hase promeed of fathid out the riff the chorus.
But the song was still not finis fo 
----
iter 280100, loss: 2.134445
----
  song was still and emen we Bric theal ring reone — lithe song was tove”.
Many im in a dueat the chotutei— ligabely bast. Finaly Hendrixt the refar the riths in a spat hound Baker ins roupen-nee 
----
iter 280200, loss: 2.112846
----
 ng ho werk as Lonapping was they they thit the song was show. The hast a ly; the riff. (InFis a vere of Nathing grilad he Brich shts on an Claduters a veasing ist fminative shopt’s drommer eans at L 
----
iter 280300, loss: 2.076458
----
  1969 Montreude apon his stilling w’s lock-boow,

----
 erment and in’s with eririchon and Baken ly in the styl. Ind in the shome pritut we”. Mot version of Nation of “Sunshine of Your Love’s all-night writing rals. (ists thardd this inste as the p 
----
iter 283500, loss: 3.836876
----
  2014; last November a veast thring way live. Foonghale of sst Pein foot 1969 the singin’s an uhise gashinely emerif pind fazzserallirson aw gis Wilson’s toil, ancererser, shige, his be vmt wrudio 
----
iter 283600, loss: 3.782046
----
 os — riguted thuin the riff to rummer Ginger ered.”. Foote, to sho pe. Ther theil supers way to the rinf. Than frement and the producer to ed in the shoped relded came frow. To Broct is and seroly 
----
iter 283700, loss: 3.670969
----
 as born wrmoans a song and swoon Cladudinallistilisenong undrd; on plyrich dp the byron Cream’s drummer Ginger Baker — suggested playing the song the Brume, Claptonceras.
MidwaR gusk aended by a s 
----
iter 283800, loss: 3.555813
----
 out 15 musiciandl sho, tawerg the song was bring a

----
 instere and caled by the US and the chon — some a song, Baker bar, the sunsir, ho werere; nesteri“Hey on therudiore ins red.
Mageaf rubord it in New York, shordy,.g thit on tomrinixt is af ad erow 
----
iter 287000, loss: 2.559493
----
 n Janum Rowa lisply bakenong a ncared through the songon’s Rounduritusted whine tract is wns baintard vest out buested the pood insa next. Spor abo,  pelly,.. They bassinger souldertsistal verter on 
----
iter 287100, loss: 2.617240
----
 endrix — have covered in Navin caste in the prodristiguded Loninglightating a nce to the Brock live shows, from playing that the lisht by balm fe war dith treringer and his bas inher voror”. Finkt 
----
iter 287200, loss: 2.791590
----
 Iuperg ruld-ter version ry the chove”. Footads. Fisomd awads — som tower in an liaktering. A hered peyMidzane a and Iher bans on an Claptoistsus vere wmel guprule of Native Ameror gule shire, Clap 
----
iter 287300, loss: 3.176272
----
  meant to join ther, on on the studio hoppen

----
 ppening a dithed the Jimi he hos cootal salt-ped bruyris shirute, andinglyred relsalman tributhis the sky began to lightruce and stume from, s his staple of Cream’s live host ort sharidicom tied oro 
----
iter 290500, loss: 2.189893
----
 up they may bl shot in a his Hendrix — have riancered pey banddlgs — lnande Amer stidiconogg a version on 1967 as Lubleas ous but in 2014; las Luluteran sowes Wharidudins Haste a nce,. Ebaly in th 
----
iter 290600, loss: 2.143939
----
 ��s a”. a aldyong on this curuiwhon meabe ilisood its rack a Hen the s of wat A Happ, in 1968 an American soul singer by the camerond Brown warert’s dromm a complyex Bricnce and band a from his He 
----
iter 290700, loss: 2.113005
----
  played the endBrifus Lulugersoor the riff a shozs Lulute, andingly the s of laithere this “It’s dy band recore from thos tuims the riff on his Hended a Hanct Han f.g stig, tomers but by the poet  
----
iter 290800, loss: 2.086453
----
 enge. Foocg sallmern Brute-d a Henter ff we 

KeyboardInterrupt: 